src/term/mod.rs

```diff
@@ -42,6 +44,9 @@ pub struct RenderableCellsIter<'a> {
     line: Line,
     column: Column,
     selection: Option<RangeInclusive<index::Linear>>,
+    custom_cursor_colors: bool,
```

Comment:
Can this flag just be passed through to `initialize` instead of keeping it as a field on the iterator?